src/Utilities/CMakeLists.txt

```diff
@@ -21,15 +26,19 @@ set(UTILITIES
     ResourceCollection.cpp
     ProjectData.cpp
     RandomNumberControl.cpp)
-add_library(qmcutil ${UTILITIES})
+add_library(qmcutil_integrated ${UTILITIES})
```

Comment:
I dislike `qmcutil_basic` and `qmcutil_integrated` it doesn't help making more organized targets. Just like `_new`, `_basic` only adds confusion. We need something that improves the clarity.  Probably rename `qmcutil_basic` to `cxx_helper` and restrict it only to enhance c++ standard library and the dependency is restricted to cxx language and standard library.

Reply:
I'm now wondering if we should just avoid significant CMake changes in this PR and just use qmcutil. breaking up qmcutil is better on a separate PR.

Reply:
I'll give it a shot.  At one point I was having linking errors I thought due to circular dependency between qmcio and qmcutil and that's what induced this.  I found another cause for that as well.

Reply:
if I restore that target split in the future I'll go with the cxx_helper name.

Reply:
I see what you mean "circular dependency between qmcio and qmcutil". If you add the new cpp to qmcuitl, It could happen, then just add cxx_helper.

Reply:
for now we are getting away with it so I think its low priority.